src/components/hero/campaignHeroSubText.tsx

```diff
@@ -29,29 +31,37 @@ const CampaignHeroSubText = () => {
           }}
           href="https://digitalcollections.nypl.org/about#nypl_harmful_content_statement"
           aria-label="Learn more about harmful content"
+          target="_blank"
         >
           Learn more
         </DSLink>
       </Text>
       <HorizontalRule />
-      <Text color="ui.typography.body" mb="0px">
-        Featured Image:{" "}
-        <DSLink
-          color="var(--nypl-colors-ui-link-primary) !important"
-          __css={{
-            _hover: {
-              color: "var(--nypl-colors-ui-link-secondary) !important",
-            },
-            _visited: {
-              color: "var(--nypl-colors-ui-link-tertiary) !important",
-            },
-          }}
-          href="https://digitalcollections.nypl.org/items/510d47e0-cb17-a3d9-e040-e00a18064a99"
-        >
-          {" "}
-          Momoyogusa = Flowers of a Hundred Generations.{" "}
-        </DSLink>
-      </Text>
+      <Tooltip content={featuredItem.title} shouldWrapChildren>
+        <Box>
+          <Text color="ui.typography.body" mb="0px" noOfLines={2}>
+            Featured Image:{" "}
+            {/* <Tooltip content={featuredItem.title} shouldWrapChildren> */}
```

Comment:
@EdwinGuzman for our 1:1, idk why but whenever I add the tooltip here it breaks the link text onto a new line.